src/rnodeAPI.js

```diff
@@ -39,24 +39,20 @@ type DeployData = {
   sig: Uint8Array,
   sigAlgorithm: string,
   from: string,
-  phloPrice: PhloPrice,
-  phloLimit: PhloLimit,
+  phloPrice: number,
+  phloLimit: number,
   nonce: number
 }
 type DeployDataInsecure = {
   term: string,
   timestamp: number,
   from: string,
-  phloPrice: PhloPrice,
-  phloLimit: PhloLimit,
+  phloPrice: number,
+  phloLimit: number,
   nonce: number
```

Comment:
Should we call this version 0.7.2 since this change is not present in the release version of 0.7.1?

I vote yes.

Reply:
I created a `node8` branch and re-targeted this pull request to that branch.